src/status_im/transport/message/contact.cljs

```diff
@@ -21,11 +21,3 @@
   (validate [this]
     (when (spec/valid? :message/contact-update this)
       this)))
-
-(fx/defn remove-chat-filter
```

Comment:
when did this function became useless? shouldn't we be removing chat-filter when we delete and/or block a contact?

Reply:
I believe this function was used when we used a symmetric key for each contact ( we don't anymore), so it's not used currently